utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import math
+import os
+import random
```

Comment:
Do you need random?

Reply:
Was already imported in the code taken from Hackerrank.